manager/orchestrator/services.go

```diff
@@ -230,5 +245,12 @@ func getRunnableSlots(s *store.MemoryStore, serviceID string) (map[uint64]slot,
 		}
 	}
 
-	return runningSlots, nil
+	deadSlots := make(map[uint64]slot)
```

Comment:
Shouldn't be a problem in practice, as we don't create tasks like this. But I changed it to only do this for tasks with `DesiredState > RUNNING`.